packages/carbon-graphs/dev-site/serve/serve-static-cli.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+const commander = require('commander');
+const serve = require('./serve-static');
+
+// Parse process arguments
+commander
+  .option('--port [number]', 'The port the app should listen on. Default: 8080', undefined)
```

Comment:
correct me if i am wrong, i think right now app is listening to 9991 by default right?

Reply:
Yes it is. Let me look at this line.

Reply:
Looks like the default is `8080` in the code:
<img width="575" alt="CleanShot 2020-10-27 at 10 12 52@2x" src="https://user-images.githubusercontent.com/38024451/97321573-06239900-183d-11eb-9b1d-a8337aebbf0d.png">